src/app/components/BibPage/BibDetails.jsx

```diff
@@ -121,9 +122,9 @@ class BibDetails extends React.Component {
       return {
         term: fieldLabel,
         definition: (
-          <ul>
-            {bibValues.map((value, i) => <li key={i}>{value}</li>)}
-          </ul>
+          <span>
```

Comment:
I don't like this spanspanspan situation... not sure what the design or design-toolkit will have in mind for styling this data though, and React wants me to wrap everything in an HTML element.